apps/air-discount-scheme/backend/src/app/modules/flight/dto/viewModel.dto.ts

```diff
@@ -30,7 +30,22 @@ export class FlightLegViewModel {
   @ApiProperty()
   readonly date: Date
 }
+export class CheckFlightViewModel {
+  constructor(status: string) {
+    if (status === '200') {
+      this.message = 'OK'
+      this.statusCode = status
+    } else {
+      // Should not end here, CheckFlight should throw an exception instead
+      this.message = 'Something went wrong'
+      this.statusCode = '500'
+    }
+  }
 
+  @ApiProperty()
+  readonly message: string
+  readonly statusCode: string
```

Comment:
isn't the statusCode an ApiProperty as well?

Reply:
:+1: